### ISSUE TITLE
Help  with connect timeout

### ISSUE THREAD

**Issue by DishDawgGaming** [GHMatti/ghmattimysql#53]
![Capture](https://user-images.githubusercontent.com/89885780/131597284-567fb3ab-91b0-436e-a67b-0e26789c42fe.PNG)
 this is the error code i am getting with a brand new qb-core  server. it is currently the stock server off of fivem.

**Comment by Cdeb123** [comment]
Yes, I am having this same issue. I have done everything correctly. One thing I did notice is that when I restart the resource after starting all resources for my server it seems to fix itself under that resource restart.  I have noticed that when you continue without doing this step and keep the resource as is, yes you may be able to still play but your database wont work properly. For example, I went without it and it messed up my housing system. I would have to restart qb-houses all the time in order to get /enter to work.

Here is how to restart this resource within txadmin
step 1: Open Resources under the left pane
step 2: search ghmattimysql 
step 3: Click on restart
Final step: go to your live console and see that it says database connected in green and you should be all good to go.

**Comment by Cdeb123** [comment]
I have the same issue, there is one thing I did that fixed it I think. I deleted ConnectQueue and re enabled hardcap and it seemed to work.
